Proctor_Brad/ruby_tdd/bankaccount/bankaccount_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'bankaccount'
+
+RSpec.describe BankAccount do
+    before do
+        @account = BankAccount.new
+    end
+    it 'has a method for retrieving the checking account balance' do
+        expect(@account.show_checking).to eq("Amount in checking: 500")
+    end
+
+    it 'has a method that retrieves the total account balance' do
+        expect(@account.show_funds).to eq("Amount in checking and savings: 1500")
+    end
+
+    it 'has a method that allows the user to withdraw cash' do
+        savings = @account.withdraw(100,'savings')
+        expect(savings).to eq(900)
+    end
+
+    it 'shows Insufficient funds when user tries to take too much from account' do
+        savings = @account.withdraw(1200,'savings')

```

Comment:
How do you know the current balance of the account? Tests run in any order and looking to understand what 1200 is in other tests makes the test harder to understand. Tests should be able to be read all by themselves. Maybe get the balance before withdrawing?